### DISCUSSION TITLE
Dax ScriptDom parser

### DISCUSSION THREAD

**Discussion by sgiovinetti** [DaxStudio/DaxStudio#790] [General]
Hello,
do you have a library that allows to parse and validate a DAX string?

I'm searching for something like this https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.transactsql.scriptdom?view=sql-dacfx-150 but for DAX instead of SQL.

Thank you in advance for your advice.

**Comment by dgosbell** [comment]
No sorry, I don't have anything like this at the moment.
